### PR TITLE
Update ContactTracingBillOfRights.md

### PR DESCRIPTION
**Pull request by danamlewis**

### CODE REVIEW

**Comment by burdges** [comment]
We'd want public epidemiological models that justify the contact tracing's effectiveness too, although not sure if that goes here.

If your infection rate stays too high, then contact tracing cannot really improve the situation.  In fact, contact tracing would *cause* infections and deaths by providing individuals, companies, and governments with an excuse to ignore other epidemiological advice.  

In Switzerland, there is one non-government organization talking about contact tracing once the entire country drops below 25 cases per day.  It's possible they are being conservative, but a denser area like NYC require even smaller numbers.  

There are never public discussions about such thresholds in the media in the U.S. and U.K.  however.  Instead, those countries have placed contact tracing into their media pump cycle, meaning they make contact tracing into an excuse to ignore other economic restrictions.

In short, contact tracing can both save lives in countries with more "responsible" governments, like South Korea or Germany, while simultaniously costing far more lives than it saves in "irresponcible" countries like the U.S. and U.K. 

All contact tracing schemes require risking medical privacy, but privacy preserving ones like TCN preserve other freedoms, like association.  I think epidemiological models being public appears necessary for people to know if they're risking medical privacy to save lives, or because they value the economy more than lives.
